app/src/main/java/com/lagradost/quicknovel/util/Apis.kt

```diff
@@ -27,6 +27,7 @@ class Apis {
             //MNovelFreeProvider(), // same as NovelFullVipProvider
             NovelFullVipProvider(),
             EngNovelProvider(),
+            NovelsOnlineProvider(),
```

Comment:
The icon might look a bit wonky with both text and icon, consider cropping the png to just include the actual icon